managed_context/metadata.json

```diff
@@ -0,0 +1 @@
+{"current_schema_version":"0.0.1"}
```

Comment:
Can you explain why you made this change please?

Reply:
@alialfie  It was a mistake, I must delete it but I can´t because the pr is closed